spec/system/admin/bulk_order_management_spec.rb

```diff
@@ -671,7 +671,7 @@
             check "bulk"
           end
           page.driver.accept_modal :confirm do
-            find("div#bulk-actions-dropdown").click   
+            find("div#bulk-actions-dropdown").click
```

Comment:
I would have done these clean-up changes in a separate commit.

Reply:
I think this came up when I ran rubocop locally. Seems like it was only this file that needed the edit